src/realm/object-store/sync/async_open_task.cpp

```diff
@@ -108,15 +108,21 @@ void AsyncOpenTask::unregister_download_progress_notifier(uint64_t token)
         m_session->unregister_progress_notifier(token);
 }
 
-void AsyncOpenTask::attach_to_subscription_initializer(AsyncOpenCallback&& callback,
-                                                       std::shared_ptr<_impl::RealmCoordinator> coordinator,
-                                                       bool rerun_on_launch)
```

Comment:
So it seems we are going to ignore the rerun_on_launch flag. Don't we have to remove it from the configuration. 
`    bool rerun_init_subscription_on_open{false};` ... also isn't this going to be a breaking change for the SDKs? Am I wrong?

Reply:
>So it seems we are going to ignore the rerun_on_launch flag.

Not entirely. We do use it in the RealmCoordinator to decide if the subscription initializer callback is invoked or not.